Add blog post about Kyma birthday (#367)
kyma-project/website: content/blog-posts/2019-07-24-happy-birthday-kyma/index.md

---
title: "Happy Birthday Kyma"
author:

>   name: "Lukasz Gornicki, Community Manager and Core&Supporting Capability Owner"

Reviewer: ```suggestion
  name: "Lukasz Gornicki, Community Manager and Core & Supporting Capability Owner"
```

tags: 
  - community
redirectFrom:
  - "/blog/happy-birthday-kyma"
---

> Kyma is already 1 year old and we have a feeling it was open sourced just recently. Let me take you for a journey in the past, let us do some flashbacks in to given time in the past to the most interesting moments. Once we know how it was in the past, let's see what future brings.

Reviewer: ```suggestion
Kyma is already 1 year old and we have a feeling it has only recently been open-sourced. Let me take you on a journey to the past to bring back the most interesting moments. Once we looked back at past events, let's see what the future holds.
```

Open source was a new land for us. When you make a step into the world of unknowns, better make it with a friend. Luckily, we were not the only ones that were at the beginning of the journey. We made an official launch at the Google Cloud Next event to be there together with the [Knative](https://github.com/knative) project.

![google-next](./google-next.png)

## 13.08.2018 Kyma made it to the CNCF Landscape

Our [`kyma`](https://github.com/kyma-project/kyma/) repository quickly reached 500 stars on GitHub. We also showed up on [CNCF Landscape](https://github.com/cncf/landscape).

![landscape-view](./landscape-screen.png)

We are definitely not planning to stop at 500 stars. This number was growing pretty slow since last year, but it has been increasing steadily since the 1.0 release. Remember that you can also star our project if you like it.

<iframe src="https://ghbtns.com/github-btn.html?user=kyma-project&repo=kyma&type=star&count=true&size=large" frameborder="0" scrolling="0" width="158px" height="30px"></iframe>

![landscape](./kyma-landscape.png)

## 12.09.2018 First Core Special Interest Group meeting

> After modeling an open governance model, we hosted our first public Core Special Interest Group meeting. It was hosted by our stars, [Piotr Kopczynski](https://twitter.com/PiotrasPk) and [Mateusz Szostok](https://twitter.com/m_szostok). 

Reviewer: ```suggestion
After creating an open governance model, we hosted our first public Core Special Interest Group meeting. It was hosted by our stars, [Piotr Kopczynski](https://twitter.com/PiotrasPk) and [Mateusz Szostok](https://twitter.com/m_szostok). 
```

Kubernetes Service Catalog is one of our main dependencies. Kyma is opinionated and declares [Service Catalog](https://github.com/kubernetes-sigs/service-catalog) and [Open Service Broker API](https://www.openservicebrokerapi.org/) as the only Service Management solution for Kubernetes that brings best developer experience. This is obvious that we are not only interested with contributing the code. We want to help to run the project and as a result [Mateusz Szostok](https://github.com/mszostok) becomes chair of the [Kubernetes Service Catalog SIG](https://github.com/kubernetes/community/tree/master/sig-service-catalog).

![chair](./chair.png)

## 03.07.2019 Setup of Knative dedicated team

Last year we went to open source with our brother in arms, [Knative](https://github.com/knative) project. Now, once we are stable and production ready, let us make this brotherhood much more dedicated. We reorganized a bit and created a team that will be dedicated to Knative contributions and then rebase Kyma on top of these contributions to improve our Event Bus and remove dependency to Kubeless.

# We make another big step

TODO

# We made friends

> We understand how important it is to make friends in open source. Leaving in isolation brings loneliness and we definitelly do not want that. Of course don't get me wrong, we are not feeling lonely as we clearly see that we have a lot of visitors of our http://slack.kyma-project.io. We make friends among users, but we also want to make friends among the projects we depend on and we see there play or will play an important role in Kyma. 

Reviewer: ```suggestion
We understand how important it is to make friends in the open-source community. Leaving in isolation brings loneliness and we definitely do not want that for our project. Fortunately, we have a lot of visitors to our main [Slack  channel](http://slack.kyma-project.io). Apart from making friends among users, we also want to make friends among the projects we depend on, especially those that play or will play an important role in Kyma. 
```

> We have few of us regularly helping the following communities:

Reviewer: ```suggestion
A few of us regularly help the following communities:
```

- [Ahmed Abdalla](https://twitter.com/Abd4llA) in [Knative](https://github.com/knative) 
- [Lukasz Gornicki](https://twitter.com/derberq) in [AsyncAPI](https://www.asyncapi.org/)
- [Piotr Kopczynski](https://twitter.com/PiotrasPk) and [Mateusz Szostok](https://twitter.com/m_szostok) in [Kubernetes Service Catalog](https://github.com/kubernetes-sigs/service-catalog)

> - [Piotr Mscichowski](https://github.com/piotrmsc) in [Ory](https://github.com/ory) and project Hydra and Authkeeper.

Reviewer: ```suggestion
- [Piotr Mscichowski](https://github.com/piotrmsc) in [Ory](https://github.com/ory), Hydra, and Authkeeper projects.
```

Reviewer: Any links, maybe?

# We contribute back

> If you depend on someone and you need more that you can get, don't wait until you get it, help out and contribute! This is a list of projects were we made noticable contributions and we are very proud of it:

Reviewer: ```suggestion
If you depend on someone and you need more than you can get, don't wait until you get it, but help out and contribute instead! This is a list of projects were we made noticeable contributions that we are very proud of:
```

> - [AsyncAPI](https://github.com/asyncapi/asyncapi-react):	Entire component donation and consulting about building community,

Reviewer: ```suggestion
- [AsyncAPI](https://github.com/asyncapi/asyncapi-react):	The entire component contribution and consultations regarding best practices for building an open-source community.
```

> - [Knative](https://github.com/knative/eventing):	Features around NATS,

Reviewer: ```suggestion
- [Knative](https://github.com/knative/eventing):	Features around NATS
```

> - [Kubernetes Service Catalog](https://github.com/kubernetes-sigs/service-catalog):	Missing features and improvements. Official approvers and chairs,

Reviewer: ```suggestion
- [Kubernetes Service Catalog](https://github.com/kubernetes-sigs/service-catalog):	Missing features and improvements; official approvers and chairs
```

> - [Open Service Broker Azure](https://github.com/Azure/open-service-broker-azure):	Missing features and improvements,

Reviewer: ```suggestion
- [Open Service Broker Azure](https://github.com/Azure/open-service-broker-azure):	Missing features and improvements
```

> - [React JSON Schema](https://github.com/mozilla-services/react-jsonschema-form):	Missing features and improvements,

Reviewer: ```suggestion
- [React JSON Schema](https://github.com/mozilla-services/react-jsonschema-form):	Missing features and improvements
```

> - [Istio](https://github.com/istio/istio):	Missing features,

Reviewer: ```suggestion
- [Istio](https://github.com/istio/istio):	Missing features
```

- [Kubeless](https://github.com/kubeless/kubeless):	Minor improvements

> That was a hell of a year, and we can't wait to see what year 2 brings, please get me some popcorn.

Reviewer: ```suggestion
That was a hell of a year, and we can't wait to see what the upcoming year holds for us!
```

# Join us

> If you were not part of our history yet, you can still join us and become part of Kyma in the future. We have many channels that you can use to interact with us:

Reviewer: ```suggestion
If you did not have a chance to contribute to our history, you can still join us and become part of Kyma in the future. We have many channels that you can use to interact with us:
```

- Twitter: https://twitter.com/kymaproject
- Core SIG public meetings: https://docs.google.com/document/d/1vWleTon7sJIk0teee4SoVezS4mR3K8TlkvXkgLJwRD8/
- Slack: http://slack.kyma-project.io
- GitHub issues: https://github.com/kyma-project/kyma/issues
- LinkedIn: https://www.linkedin.com/company/kyma-project/
- Stackoverflow: https://stackoverflow.com/questions/tagged/kyma
- YouTube: https://www.youtube.com/channel/UC8Q8bBtYe9gQN-dQ-_L8JvQ

> Choose the channel that is the most cozy for you and let us know what you think about Kyma and ask how you can start.

Reviewer: ```suggestion
Use one of these communication channels to let us know what you think about Kyma and how you can start your contribution to the project.
```

> Choose the channel that is the most cozy for you and let us know what you think about Kyma and ask how you can start.

Reviewer: ```suggestion
Use one of these communication channels to let us know what you think about Kyma and how you can start your contribution to the project.
```

  * [19.02.2019 First Kyma early adopters](#19-02-2019-first-kyma-early-adopters)
  * [25.04.2019 1.0 release](#25-04-2019-1-0-release)
  * [02.05.2019 Netconomy went to production](#02-05-2019-netconomy-went-to-production)
  * [07.05.2019 Kyma released with SAP C/4HANA Foundation to the production environment](#07-05-2019-kyma-released-with-sap-c-4hana-foundation-to-the-production-environment)
  * [21-23.05.2019 Kyma at KubeCon Barcelona](#21-23-05-2019-kyma-at-kubecon-barcelona)
  * [23.05.2019 We announced Kyma 1.0 on the Kubernetes blog](#23-05-2019-we-announced-kyma-1-0-on-the-kubernetes-blog)
  * [18.06.2019 Kyma devs became Kubernetes contributors and Service Catalog approvers](#18-06-2019-kyma-devs-became-kubernetes-contributors-and-service-catalog-approvers)
  * [02.07.2019 Kyma dev became a chair of the Service Catalog SIG](#02-07-2019-kyma-dev-became-a-chair-of-the-service-catalog-sig)
  * [03.07.2019 Setup of Knative dedicated team](#03-07-2019-setup-of-knative-dedicated-team)
- [We make another big step](#we-make-another-big-step)
- [We made friends](#we-made-friends)
- [We contribute back](#we-contribute-back)

> - [Join us](#join-us)

Reviewer: ```suggestion
- [We want you to contribute - birthday gifts](#we-want-to-contribute---birthday-gifts)
- [Join us](#join-us)
```

# We contribute back

If you depend on someone and you need more than you can get, don't wait until you get it, but help out and contribute instead! This is a list of projects were we made noticeable contributions that we are very proud of:

- [AsyncAPI](https://github.com/asyncapi/asyncapi-react):	The entire component contribution and consultations regarding best practices for building an open-source community.
- [Knative](https://github.com/knative/eventing):	Features around NATS
- [Kubernetes Service Catalog](https://github.com/kubernetes-sigs/service-catalog):	Missing features and improvements; official approvers and chairs
- [Open Service Broker Azure](https://github.com/Azure/open-service-broker-azure):	Missing features and improvements
- [React JSON Schema](https://github.com/mozilla-services/react-jsonschema-form):	Missing features and improvements
- [Istio](https://github.com/istio/istio):	Missing features
- [Kubeless](https://github.com/kubeless/kubeless):	Minor improvements

That was a hell of a year, and we can't wait to see what the upcoming year holds for us!

# We want you to contribute - birthday gifts

> Birthdays are always connected with presents. This time we plan to give away some Kyma t-shirts. Not for free thought, you get those only if you become an external contributor in next 7 days. Our undestanding of an external contributor is that you are not a part of GitHub Kyma organization and during next 7 days you:

Reviewer: ```suggestion
Birthdays are always connected with presents. This time we plan to give away some Kyma t-shirts, not for free thought. You get those only if you become an external contributor in the next 7 days. Our understanding of an external contributor is that you are not a member of the Kyma organization on GitHub, and for the next 7 days you:
```

> - Contributed code or documentation to one of Kyma repositories. It must be a pull request, but not necesarily merged as we can imagine that depending on the size of the contribution it might not be that easy to approve and merge it.

Reviewer: ```suggestion
- Contributed code or documentation to one of Kyma repositories. It must be a pull request, but not necessarily merged. We can imagine that depending on the size of the contribution it might not be that easy to approve and merge it.
```

> - Contributed an issue using one of our templates. We value equaly a well described bug that is easy to replicate and an enhancement request with well described use case.

Reviewer: ```suggestion
- Contributed an issue using one of our templates. We equally value a well-described bug that is easy to replicate and an enhancement request with a well-described use case.
```

> All Kyma repositories are located here: https://github.com/kyma-project/kyma/

Reviewer: ```suggestion
All Kyma repositories are located [here](https://github.com/kyma-project/kyma/).
```

> Once you contributed, let us know about it on [Twitter](https://twitter.com/kymaproject) or [Slack](http://slack.kyma-project.io), the #general channel. We will evaluate all contributions on Thursday, 1.08 and communicate all accepted contributions and happy winners. Happy contributing!

Reviewer: ```suggestion
Once you contribute, let us know about it on [Twitter](https://twitter.com/kymaproject) or [Slack](http://slack.kyma-project.io), on the **#general** channel. We will evaluate all contributions on Thursday, August 1, and communicate all accepted contributions and happy winners. Happy contributing!
```

> You don't know what to contribute? I'm sure we can help, just ping us on [Twitter](https://twitter.com/kymaproject) or [Slack](http://slack.kyma-project.io), the #general channel.

Reviewer: ```suggestion
If you don't know what to contribute, I'm sure we can help - just contact us through one of the above communication channels and we will point you in the right direction.
```

  * [07.05.2019 Kyma released with SAP C/4HANA Foundation to the production environment](#07-05-2019-kyma-released-with-sap-c-4hana-foundation-to-the-production-environment)
  * [21-23.05.2019 Kyma at KubeCon Barcelona](#21-23-05-2019-kyma-at-kubecon-barcelona)
  * [23.05.2019 We announced Kyma 1.0 on the Kubernetes blog](#23-05-2019-we-announced-kyma-1-0-on-the-kubernetes-blog)
  * [18.06.2019 Kyma devs became Kubernetes contributors and Service Catalog approvers](#18-06-2019-kyma-devs-became-kubernetes-contributors-and-service-catalog-approvers)
  * [02.07.2019 Kyma dev became a chair of the Service Catalog SIG](#02-07-2019-kyma-dev-became-a-chair-of-the-service-catalog-sig)
  * [03.07.2019 Setup of Knative dedicated team](#03-07-2019-setup-of-knative-dedicated-team)
- [We are making another big step](#we-are-making-another-big-step)
- [We made friends](#we-made-friends)
- [We contribute back](#we-contribute-back)
- [We want you to contribute - birthday gifts](#we-want-you-to-contribute---birthday-gifts)
- [Join us](#join-us)

# We have had quite an interesting year

## 24.07.2018 Open source - day one

> We know what we did last summer, and we are very proud of it. We opened our repositories to the outside world and published the first version of our website. We introduced our project in a [blog post](https://kyma-project.io/blog/2018/7/24/introduction-project-kyma/] and took first steps on [Twitter](https://twitter.com/kymaproject), growing to over 700 followers.

Reviewer 2: This file fails in Prow. It should be `[blog post]((https://kyma-project.io/blog/2018/7/24/introduction-project-kyma/)`, not with `]`
